enterprise/metrics/src/main/java/org/neo4j/metrics/source/ClusterMetrics.java

```diff
@@ -56,6 +72,7 @@ public void start() throws Throwable
     {
         if ( config.get( MetricsSettings.neoClusterEnabled ) )
```

Comment:
this should be `if ( config.get( MetricsSettings.neoClusterEnabled && resolveClusterMembersDependencyOrLogWarning )`